data/templates/src/elf/dll/elf_dll_x64_template.s

```diff
@@ -77,7 +74,7 @@ dynsection:
   dq    strtab
 ; DT_STRSZ
   dq    0x0a
-  dq    strtabsz
```

Comment:
The string/symbol table is technically empty and should have a size of 0. The two null bytes in the table should only be used by the `sh_name` references. This causes no problems but was technically incorrect.